--- squid-4.4/debian/rules.clean	2019-05-17 18:51:54.485788597 +0000
+++ squid-4.4/debian/rules	2019-05-17 18:01:43.522023353 +0000
@@ -23,6 +23,9 @@
 
 BUILDINFO := $(shell lsb_release -si 2>/dev/null)
 
+override_dh_shlibdeps:
+	dh_shlibdeps --dpkg-shlibdeps-params=--ignore-missing-info
+
 DEB_CONFIGURE_EXTRA_FLAGS := BUILDCXXFLAGS="$(CXXFLAGS) $(CPPFLAGS) $(LDFLAGS)" \
 		--enable-build-info="$(BUILDINFO) $(DEB_HOST_ARCH_OS)" \
 		--datadir=/usr/share/squid \
@@ -36,7 +39,7 @@
 		--enable-removal-policies="lru,heap" \
 		--enable-delay-pools \
 		--enable-cache-digests \
-		--enable-icap-client \
+		--enable-icap-client --enable-ssl --enable-ssl-crtd --with-openssl=/opt/libressl \
 		--enable-follow-x-forwarded-for \
 		--enable-auth-basic="DB,fake,getpwnam,LDAP,NCSA,NIS,PAM,POP3,RADIUS,SASL,SMB" \
 		--enable-auth-digest="file,LDAP" \